velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +91,23 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct InverseBetaCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(double& result, double a, double b, double p) {
+    static constexpr double kInf = std::numeric_limits<double>::infinity();
+
+    VELOX_USER_CHECK((a > 0) && (a != kInf), "a must be a positive real value");
```

Comment:
Please use the same error string messages and order of the errors as in Presto java code at 
https://github.com/prestodb/presto/blob/37a05171308cde8478a3bced0bc51613afaa64de/presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java#L826

The check for p happens before a and b and the error messages are different.

Reply:
Addressed.